test/nuxt/composables.test.ts

```diff
@@ -923,6 +923,32 @@ describe('useFetch', () => {
     await nextTick()
     expect.soft(status.value).toBe('pending')
   })
+
+  it.skipIf(process.env.NUXT_LEGACY !== '1')('should handle parallel execute with `immediate: false`', async () => {
```

Comment:
I updated the env variable:

```suggestion
  it.runIf(process.env.PROJECT === 'nuxt-legacy')('should handle parallel execute with `immediate: false`', async () => {
```